airflow/www/utils.py

```diff
@@ -682,6 +684,12 @@ def get_attr_renderer():
     }
 
 
+def get_trimmed_field(query: Query, model: Model, column_name: str) -> tuple[Query, ColumnElement]:
+    query, field = get_field_setup_query(query, model, column_name)
+    trimmed_value = func.btrim(func.convert_from(field, "UTF8"), '"')
+    return query, trimmed_value
```

Comment:
another idea I have is doing something like this

```suggestion
def generate_filter_value_query(
    *,
    query: Query,
    model: Model,
    column_name: str,
    filter_cond: Callable[str, Query]
) -> Query
    query, field = get_field_setup_query(query, model, column_name)
    trimmed_value = func.btrim(func.convert_from(field, "UTF8"), '"')
    return query.filter(filter_cond(trimmed_value))
```